api/test/factories/factory_helper.js

```diff
@@ -157,7 +156,7 @@ function hexaDecimal(count) {
 
 function generateSeededObjectId(value) {
     let oid = (typeof value === "undefined") ? hexaDecimal(24).toLocaleLowerCase() : value;
-    if (!bsonObjectId.isValid(oid)) throw "Invalid attempt to generate an ObjectID: '" + oid + "'";
+    if (!bsonObjectId.isValid(oid)) throw new Error("Invalid attempt to generate an ObjectID: '" + oid + "'");
```

Comment:
I often find it a bit cleaner to use a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) over concatenations

```if (!bsonObjectId.isValid(oid)) throw new Error(`Invalid attempt to generate an ObjectID: '${oid}'`);```